src/Relewise.Client.Extensions/RelewiseClientOptions.cs

```diff
@@ -7,13 +7,16 @@ namespace Relewise.Client.Extensions;
 /// </summary>
 public class RelewiseClientOptions : IEquatable<RelewiseClientOptions>
 {
+    private const string BaseProductionServerUrl = "https://api.relewise.com";
+
     /// <summary>
     /// Initializes a new instance of the <see cref="RelewiseClientOptions"/>.
     /// </summary>
-    /// <param name="datasetId">Defines the id of the dataset the client should access. The value can be found in https://my.relewise.com.</param>
-    /// <param name="apiKey">Defines the api key that should be used. Api keys can be found (and created) in https://my.relewise.com.</param>
+    /// <param name="datasetId">Defines the id of the dataset the client should access. The value can be found at https://my.relewise.com/developer-settings.</param>
+    /// <param name="apiKey">Defines the api key that should be used. Api keys can be found (and created) at https://my.relewise.com/developer-settings.</param>
     /// <param name="timeout">Defines the timeout to be used by the client.</param>
-    public RelewiseClientOptions(Guid datasetId, string apiKey, TimeSpan timeout)
+    /// <param name="serverUrl">Defines the url of the server to target.The value can be found at https://my.relewise.com/developer-settings.</param>
+    public RelewiseClientOptions(Guid datasetId, string apiKey, TimeSpan timeout, string? serverUrl)
```

Comment:
```suggestion
    public RelewiseClientOptions(Guid datasetId, string apiKey, TimeSpan timeout, string? serverUrl = null)
```

Det er ikke påkrævet at angive denne

Reply:
Check! 🙂